### PR TITLE
DEV-4941: bloc-counter audit and readme

### PR DESCRIPTION
**Pull request by MSilva95**
fix bloc-counter audit and readme

### CODE REVIEW

File: subjects/mobile-dev/bloc-counter/audit/README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
 ###### Does the app run without any crashes?
 
-###### Can you increment counter?
+###### Can you increment the counter by tapping the "+" button?
 
-###### Can you decrement counter?
+###### Can you decrement the counter by tapping the "-" button?
```
Comment (nprimo) [inline subjects/mobile-dev/bloc-counter/audit/README.md:9]: In the subject, decrementing is described as a bonus

File: subjects/mobile-dev/bloc-counter/audit/README.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 
-###### Is the BLoC pattern implemented properly? Do you see blockobserver, blockprovider, blockbuilder explicitly?
+###### Is the BLoC pattern implemented properly? Can you see the BlocObserver, BlocProvider, and BlocBuilder being used explicitly in the code?
 
-###### Are incrementing and decrememting implemented using BLoC pattern?
+###### Are the increment and decrement operations implemented using the BLoC pattern, with the CounterBloc class managing the state of the counter?
```
Comment (nprimo) [inline subjects/mobile-dev/bloc-counter/audit/README.md:13]: In the subject, decrementing is described as a bonus

**Review by nprimo — CHANGES_REQUESTED** [review]
The subject and audit are inconsistent regarding the feature "decrementing": in the subject, it is described as a bonus while in the audit it seems to always be expected.

File: subjects/mobile-dev/bloc-counter/audit/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/mobile-dev/bloc-counter/audit/README.md]: done

File: subjects/mobile-dev/bloc-counter/audit/README.md (hunk 2 above)
Comment (MSilva95) [reply subjects/mobile-dev/bloc-counter/audit/README.md]: done

**Review by nprimo — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**